src/server/game/Entities/Player/Player.cpp

```diff
@@ -22138,6 +22138,35 @@ void Player::SendCooldownEvent(SpellInfo const* spellInfo, uint32 itemId /*= 0*/
     data << uint32(spellInfo->Id);
     data << uint64(GetGUID());
     SendDirectMessage(&data);
+
+    uint32 cat = spellInfo->GetCategory();
+    uint32 catRec = spellInfo->CategoryRecoveryTime;
```

Comment:
line 22143 is useless remove it 
uint32 catRec = spellInfo->CategoryRecoveryTime;